Sources/Search/OASearchCoreFactory.mm

```diff
@@ -752,6 +751,10 @@ - (BOOL) search:(OASearchPhrase *)phrase resultMatcher:(OASearchResultMatcher *)
         if (!r)
             continue;
         searchCriteria->localResources = {r};
+        if ([resId containsString:@"basemap"])
+            searchCriteria->bbox31 = OsmAnd::AreaI(0, 0, INT_MAX, INT_MAX);
```

Comment:
`bbox31` is `Nullable`, thus it is better to use this property instead of `OsmAnd::AreaI(0, 0, INT_MAX, INT_MAX)`